src/client/client.spec.ts

```diff
@@ -99,11 +99,13 @@ describe("Client:", () => {
       expect(numOfKeysBefore + 1).toBe(numOfKeysAfter);
     });
 
-    it("should not be able to overwrite a possible event", () => {
+    it('should not be able to overwrite a possible event', () => {
       expect(client._events.hasOwnProperty('testevent')).toBe(false);
 
       client.addEvents('testevent');
-      client.on('testevent', function() {});
+      client.on('testevent', function() {
+        console.log('testevent');
```

Comment:
is this so it does not yell at you for having an empty function? because this console.log is not required as it only checking to see if the property is actually there and not what the data is

Reply:
yep just to remove a error